tests/integ/modin/frame/test_merge.py

```diff
@@ -92,9 +92,8 @@ def _merge_native_pandas_frames_on_index_on_both_sides(
     is_left_mi = left.index.nlevels > 1
     is_right_mi = right.index.nlevels > 1
     if sort:
-        # When joining single index frame with multi-index or multi-index frame with
-        # single index, native pandas doesn't respect 'sort' argument. It always
-        # behaves as sort=False.
+        # When multi-index is involved on either (or both) side of frames, native pandas
```

Comment:
The comment says "or both", but the condition doesn't seem to include the both case: `(is_left_mi and not is_right_mi) or (not is_left_mi and is_right_mi)`

Reply:
It's the second if block below `is_left_mi and is_right_mi` based on if both sides have multi index or not sorting steps are different.